docs/ref/settings.txt

```diff
@@ -385,6 +385,15 @@ where ``reason`` is a short message (intended for developers or logging, not for
 end users) indicating the reason the request was rejected.  See
 :doc:`/ref/csrf`.
 
+The default implementation ``django.views.csrf.csrf_failure`` takes an additional
+keyword parameter ``template_name`` that defaults to ``'403_csrf.html'``. If a
+template can be found under this name, it will be used to render the error
+message.
+
+.. versionchanged:: 1.10
+
+   The ``template_parameter`` of ``csrf_failure`` has been added.
```

Comment:
Sure, fixed. That's what happs when one starts working on two different things at the same time, I guess…